### PR TITLE
Add new NUI samples

### PR DESCRIPTION
**Pull request by sgchoi5**
Added new NUI samples - 13 for Mobile and 1 for TV

### CODE REVIEW

**Comment by sgchoi5** [comment]
Looks good to me

File: docs/application/dotnet/samples/ui/overview.md [hunk 1]
```diff
@@ -99,6 +99,99 @@ for TD:
...
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html" target="_blank">VideoView</a>.<br/>
+			It performs several functionalities like forward, backward, play, pause, or stop a video.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m70view.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/View" target="_blank"><strong>(M) View (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html" target="_blank">View</a>.<br/>
+			Press "Next" to see different View configurations.</p>
```
Comment (nawab-samsung) [inline docs/application/dotnet/samples/ui/overview.md:185]:
```suggestion
			Tap **Next** to see different View configurations.</p>

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@sgchoi5 A slight modification needed, request you to incorporate the comment.

File: docs/application/dotnet/samples/ui/overview.md (hunk 1 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: Thanks!. fixed it

Comment (nawab-samsung) [reply docs/application/dotnet/samples/ui/overview.md]: @sgchoi5 my intention was to make it bold, I thought md tag will work but this being a HTML file. 

```suggestion
			Tap <b>Next</b>to see different View configurations.</p>

File: docs/application/dotnet/samples/ui/overview.md [hunk 2]
```diff
@@ -99,6 +99,99 @@ for TD:
...
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html" target="_blank">VideoView</a>.<br/>
+			It performs several functionalities like forward, backward, play, pause, or stop a video.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m70view.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/View" target="_blank"><strong>(M) View (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html" target="_blank">View</a>.<br/>
+			Press **Next** to see different View configurations.</p>
```
Comment (nawab-samsung) [inline docs/application/dotnet/samples/ui/overview.md]: @sgchoi5 
My intention was to make it bold, I thought md tag will work as it renders html when built, but in this case it's not, so using HTML tag will do it for us. 

```suggestion
			Tap<b>Next</b>to see different View configurations.</p>

File: docs/application/dotnet/samples/ui/overview.md (hunk 1 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: Thanks!. Fixed it!

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@sgchoi5 Request you to incorporate the comment, rest looks good. Shall approve the PR afterwards.

File: docs/application/dotnet/samples/ui/overview.md (hunk 2 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: Thanks! Fixed it!

**Review by nawab-samsung — APPROVED** [review]
@sgchoi5 Thanks !

**Comment by mijin85cho** [comment]
Hello, @annie-samsung 
Would you please check this PR and proceed?

**Pull request merged**